docs/pages/components/dialog.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Dialog
+keywords: dialog, dialogs
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/dialog.html
+folder: components
+summary:
+---
+
+The dialog is a container generally displayed in response to an action. It is used for short forms, confirmation messages or to display contextual information that does not require a page. The dialog should always be used in conjunction with the [Application Layout Containers](/layouts/application-layout.html#application-with-ui-overlay). See an example [App layout page with Modal](/demo-pages/modal-overlay-demo-page.html)
+
+<hr/>
+
+## Informational Dialog
+This is used to present information to the user.
+
+{% capture default-dialog %}
+<div class="modal-demo-bg">
+<div class="fd-dialog--small" role="dialog">
```

Comment:
There shouldn't be a specific `small` size. The size is only to indicate the size on phone. This should be done with `max-width` and media queries.